### PR TITLE
[WIP] Problem: We need a demonstration of execution

### PR DESCRIPTION
**Pull request by rgbkrk**
Solution: Show a WIP approach to running code.

screenshot 2016-02-06 22 06 34

This is missing cleanup, mapping responses to notebook format, etc. It makes me realize I should continue on with a smaller PR (library?) that would handle merging the responses and outputs into the cell's notebook structure.

### CODE REVIEW

**Comment by jdfreder** [comment]
Sweet, I'm trying this locally now..

File: src/components/Cell/CodeCell.jsx [hunk 1]
```diff
@@ -3,20 +3,131 @@ import React from 'react';
...
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({
+      source: nextProps.input,
+    });
+  }
+
+  keyDown(e) {
+    if (!e.shiftKey || e.key !== 'Enter') {
```
Comment (jdfreder) [inline src/components/Cell/CodeCell.jsx:49]: For some reason this is not triggering for me...  Looking into it....

Comment (rgbkrk) [inline src/components/Cell/CodeCell.jsx:49]: All the messages appear in the chrome console here (see screenshot).

Comment (jdfreder) [inline src/components/Cell/CodeCell.jsx:49]: Not here :(

**Comment by rgbkrk** [comment]
Closing for now, will be coming back to this again while leaving my branch to look over it again.

**Pull request closed without merging**